dist/scss/mixins.scss

```diff
@@ -49,6 +49,14 @@
   -webkit-transform:rotate($angle); /* Opera, Chrome, and Safari */
 }
 
+@mixin animation($str) {
+    -webkit-animation: #{$str};
+    -moz-animation: #{$str};
+    -ms-animation: #{$str};
+    -o-animation: #{$str};
+    animation: #{$str};
+}
```

Comment:
Do we have to implement that on our own? Can't we use the [compass mixin](http://compass-style.org/reference/compass/css3/animation/)?